css/style.scss

```diff
@@ -0,0 +1,382 @@
+@import "fonts";
+@import "slider";
+
+
+$mainBgColor: rgb(45, 48, 58);
+$mainBgColorBorder: rgb(50, 55, 70);
+$redColor: rgb(255, 108, 100);

```

Comment:
What will be if we decide to use, not red colour as active?

Reply:
Now renamed $mainColor